AlphaWallet/Core/NFT/Enjin/Enjin.swift

```diff
@@ -48,7 +46,7 @@ final class Enjin {
                     seal.reject(error)
                 }
             }
-        }.then(on: queue, { [networkProvider] balances -> Promise<EnjinSemiFungiblesToAddress> in
+        }.then(on: .none, { [networkProvider] balances -> Promise<EnjinSemiFungiblesToAddress> in
```

Comment:
We can just do `then() {}` and let it default to the fallback queue and pass in `.global()` or a custom queue like in the comment in the other commit if it's slow.